mkdocs.yml

```diff
@@ -77,15 +77,15 @@ nav:
     - gRPC Routing: guides/grpc-routing.md
     - Migrating from Ingress: guides/migrating-from-ingress.md
   - Reference:
-    - Implementer's Guide: references/implementers-guide.md
+    - Implementer's Guide: reference/implementers-guide.md
```

Comment:
Can you add some corresponding entries to `redirect_maps` above?

Reply:
Done